littlelight.py

```diff
@@ -174,7 +184,8 @@ def charCreate(save, player):
                         charCreate(save, player)
                         player.health = player.strength * 5 + player.glow * 2
                         player.curr_health = player.health
-                        save.objects.append(player.getStats())
+                        save.objects.get('save_1').update(player.getStats())

```

Comment:
Manually assigning the save to save one seems like it would either exclude any other saves or really bloat the code. Maybe instead, have a variable that is assigned based on which save file is selected to load? i.e. on save 2 button click the variable saveName is set to 'save_2', and save_objects.get() uses saveName as the selection

Reply:
For this one, this is the call to new game. Right now theres now way to select a save game so whenever the user selects new game it would default to save 1. But will try to implement for the continue menu

Reply:
ah that makes sense, my bad